pkgs/top-level/coq-packages.nix

```diff
@@ -88,6 +88,7 @@ let
       ITree = callPackage ../development/coq-modules/ITree { };
       itree-io = callPackage ../development/coq-modules/itree-io { };
       json = callPackage ../development/coq-modules/json {};
+      jasmin-proofs = callPackage ../development/coq-modules/jasmin-proofs {};
```

Comment:
Why `jasmin-proofs`? IMHO, `jasmin` would be more appropriate.

Reply:
I guess because it's compiling the `proofs` directory. But you certainly know better than me (to be honest, jasmin is currently just a CI entry among others for me).